### PR TITLE
ui: revamp entity task component

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

- I worked on supporting task for tag and description from the single component

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Review by chirag-madlani — COMMENTED** [review]
@Sachin-chaurasiya can you have a look here?

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/EntityTasks/EntityTasks.interface.ts [hunk 1]
```diff
@@ -11,19 +11,21 @@
...
   data: {
     fqn: string;
-    tags: TagLabel[];
+    field?: string | TagLabel[];
   };
-  tagSource: TagSource;
+  tagSource?: TagSource;
   entityFqn: string;
   entityType: EntityType;
+  entityTaskType: EntityField.TAGS | EntityField.DESCRIPTION;
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/EntityTasks/EntityTasks.interface.ts:28]: Can't we use the `EntityField` enum directly?

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/EntityTasks/EntityTasks.test.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+// import { EntityField } from 'constants/Feeds.constants';
+// import { FQN_SEPARATOR_CHAR } from 'constants/char.constants';
+// import { DE_ACTIVE_COLOR } from 'constants/constants';
+// import { ENTITY_TASKS_TOOLTIP } from 'constants/entity.constants';
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/EntityTasks/EntityTasks.test.tsx:17]: Remove this.

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/EntityTasks/EntityTasks.test.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  it('Task Element should be visible when tagSource is not glossary', async () => {
+    render(<EntityTasks {...mockProps} />);
+
+    const container = await screen.findByTestId('entity-task');
+
+    expect(container).toBeInTheDocument();
+
+    const taskElement = screen.queryByTestId('task-element');
+
+    screen.debug(container);
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/EntityTasks/EntityTasks.test.tsx:85]: remove this.

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/EntityTasks/EntityTasks.test.tsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+jest.mock('../../../utils/CommonUtils', () => ({
+  getPartialNameFromTableFQN: jest.fn().mockReturnValue('test'),
+}));
+
+jest.mock('react-router-dom', () => ({
+  useHistory: jest.fn(),
+}));
+
+describe('Entity Task component', () => {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/EntityTasks/EntityTasks.test.tsx:67]: Can we cover the test cases for both task types, `tags` and `description`?

**Review by Sachin-chaurasiya — COMMENTED** [review]
Left a few comments, thanks

**Comment by Ashish8689** [comment]
@Sachin-chaurasiya  changes updated as per comments!

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**